Lambda-School-Labs/labs9-stock-trainer
# Haywood/django backend

# Description

Added Django backend, app settings, and coverage for testing.

Fixes # (issue)

## Type of change


# How Has This Been Tested?

1. ensure you have pip and pipenv installed on your system
1. run `pipenv install`
1. from root, activate `pipenv shell` and run `./manage.py runserver` to see Django welcome page
1. run `coverage run --source='.' manage.py test` then `coverage html`
1. check for htmlcov to see what needs testing; already in the .gitignore


# Checklist:
